.vscode/launch.json

```diff
@@ -4,20 +4,6 @@
   // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
   "version": "0.2.0",
   "configurations": [
-    {
-      "name": "Debug Server Jest Tests",
-      "type": "node",
-      "request": "launch",
-      "runtimeArgs": [
-        "--inspect-brk",
-        "node_modules/.bin/jest",
-        "--runInBand",
-        "--config",
-        "jestconfig.json"
-      ],
-      "console": "integratedTerminal",
-      "internalConsoleOptions": "neverOpen"
-    },
     {
```

Comment:
How to debug tests?

Reply:
Use https://marketplace.visualstudio.com/items?itemName=hbenl.vscode-mocha-test-adapter extension. Actually I thought you already use it.